package.json

```diff
@@ -13,7 +13,8 @@
     "dist-all": "build -mwl --x64 --ia32",
     "pack": "build --dir",
     "dist": "build --x64 --ia32",
-    "test": "mocha test"
+    "test": "eslint . && mocha test",
```

Comment:
I think it's not the eslint issue, that's why I wanted to create 2 PRs from this. 

You can reference eg. to https://github.com/hovancik/stretchly/blob/master/.travis.yml to see that I am not using `npm test` but `istanbul cover ./node_modules/mocha/bin/_mocha`.